chapters/ch05/2.index-datasets.ipynb

```diff
@@ -138,164 +31,42 @@
-      " |-- last_editor_display_name: string (nullable = true)\n",
-      " |-- last_edit_date: timestamp (nullable = true)\n",
-      " |-- last_activity_date: timestamp (nullable = true)\n",
-      " |-- title: string (nullable = true)\n",
-      " |-- tags: string (nullable = true)\n",
-      " |-- answer_count: integer (nullable = true)\n",
-      " |-- comment_count: integer (nullable = true)\n",
-      " |-- favorite_count: integer (nullable = true)\n",
-      " |-- closed_date: timestamp (nullable = true)\n",
-      " |-- community_owned_date: timestamp (nullable = true)\n",
-      " |-- category: string (nullable = false)\n",
-      "\n",
-      "Successfully written 12892 documents\n",
-      "Loading data/cooking/posts.csv\n",
-      "Schema: \n",
-      "root\n",
-      " |-- post_type_id: integer (nullable = true)\n",
-      " |-- accepted_answer_id: integer (nullable = true)\n",
-      " |-- parent_id: integer (nullable = true)\n",
-      " |-- creation_date: timestamp (nullable = true)\n",
-      " |-- deletion_date: string (nullable = true)\n",
-      " |-- score: integer (nullable = true)\n",
-      " |-- view_count: integer (nullable = true)\n",
-      " |-- body: string (nullable = true)\n",
-      " |-- owner_user_id: integer (nullable = true)\n",
-      " |-- owner_display_name: string (nullable = true)\n",
-      " |-- last_editor_user_id: integer (nullable = true)\n",
-      " |-- last_editor_display_name: string (nullable = true)\n",
-      " |-- last_edit_date: timestamp (nullable = true)\n",
-      " |-- last_activity_date: timestamp (nullable = true)\n",
-      " |-- title: string (nullable = true)\n",
-      " |-- tags: string (nullable = true)\n",
-      " |-- answer_count: integer (nullable = true)\n",
-      " |-- comment_count: integer (nullable = true)\n",
-      " |-- favorite_count: integer (nullable = true)\n",
-      " |-- closed_date: timestamp (nullable = true)\n",
-      " |-- community_owned_date: timestamp (nullable = true)\n",
-      " |-- category: string (nullable = false)\n",
-      "\n",
-      "Successfully written 79324 documents\n",
+      "Collection [health] exists? True\n",
+      "Documents expected: 12892\n",
+      "Documents found: 12892\n",
+      "Collection [health] is healthy\n",
+      "Collection [cooking] exists? True\n",
+      "Documents expected: 79324\n",
+      "Documents found: 17647\n",
+      "Reindexing [cooking] collection\n",
       "Wiping \"cooking\" collection\n",
       "Creating \"cooking\" collection\n",
+      "{'responseHeader': {'status': 0, 'QTime': 1031}, 'success': {'aips-solr:8983_solr': {'responseHeader': {'status': 0, 'QTime': 216}, 'core': 'cooking_shard1_replica_n1'}}, 'warning': 'Using _default configset. Data driven schema functionality is enabled by default, which is NOT RECOMMENDED for production use. To turn it off: curl http://{host:port}/solr/cooking/config -d \\'{\"set-user-property\": {\"update.autoCreateFields\":\"false\"}}\\''}\n",
```

Comment:
Could we not output the actual response from the engine unless there's an error? I think this just adds noise throughout the notebooks in the normal case of successful indexing.